.github/workflows/pypi-wheel-builds.yml

```diff
@@ -1,10 +1,9 @@
-name: Spglib PyPI Wheel builds for linux
+name: pypi-wheel
+run-name: Spglib PyPI Wheel builds for linux
 
 on:
   push:
-    branches:
-      - rc
-      - master
```

Comment:
I think the `master` branch is still required to upload PyPI.

Reply:
Yes, master is needed!

Reply:
Oh. Want to change it to tags?

Reply:
> Want to change it to tags?

It is also a choice. Currently master and tag are in one-to-one correspondence in my operation.

Reply:
Mostly it's because repos tend to move away from branches named `master` and if we move to tags, it would be completely unnecessary to keep master branch up to date

Reply:
Can check my tests for the push tags on my repo:
https://github.com/LecrisUT/spglib/actions/runs/4241036443

Reply:
About this?
```
on:
  push:
    branches:
      - rc
      - master
  pull_request:
    branches: [rc]
```

Does this
```
  pull_request:
    branches: [rc]
```
mesn a trigger when PR is made to the rc branch? It seems a good idea.

Reply:
Indeed. I was wondering about how to allow contributors to test the github actions for pypi. But apparently if you push tags locally it also triggers the build. It is not ideal because it will also run the `upload_pypi` step, but it should not be able to push an actual release there because it wouldn't have the secrets (I confirmed it on my fork https://github.com/LecrisUT/spglib/actions/runs/4241036443/jobs/7371203802).

But yeah, maybe having it automatic on `rc` would be easier for the contributors. Maybe change the name to `test-pypi` and/or document it more in `Contributing.md`?

One concern and probably only @lan496 can check for us, are the gihub action minutes billed on the organization? I don't see any Github action billing usage for this repo on my billing report. This workflow does eat up a lot of minutes (45min/run)

Reply:
Current plan is GitHub Free.

Reply:
The workflow is slow because of VM emulation of different CPU architecture (arm64) for linux if I remember correctly.

Reply:
I have definitely used more than 300 minutes on this PR alone. Let's give it a few days and check again when it's updated.

Reply:
Ok. I will check it again at the end of this month.

Reply:
Oh, now I remember, the billing is only for private repos

Reply:
What generous Github!